### PR TITLE
ISPN-13192 Warning messages in xsite test logs

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.redhat.com/browse/ISPN-13192

Use separate mcast addresses and ports for each site.

### CODE REVIEW

File: core/src/test/java/org/infinispan/distribution/rehash/ConcurrentJoinTest.java [hunk 1]
```diff
@@ -28,7 +28,8 @@ void performRehashEvent(boolean offline) throws Exception {
       joiners = new CopyOnWriteArrayList<Cache<Object, String>>(new Cache[NUM_JOINERS]);
 
       for (int i = 0; i < NUM_JOINERS; i++) {
-         EmbeddedCacheManager joinerManager = addClusterEnabledCacheManager(TestDataSCI.INSTANCE, new ConfigurationBuilder(), new TransportFlags().withFD(false).withPortRange(i));
+         EmbeddedCacheManager joinerManager = addClusterEnabledCacheManager(TestDataSCI.INSTANCE, new ConfigurationBuilder(),
+                                                                            new TransportFlags().withFD(false));
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/distribution/rehash/ConcurrentJoinTest.java:32]: nit
```suggestion
         EmbeddedCacheManager joinerManager = addClusterEnabledCacheManager(TestDataSCI.INSTANCE, new ConfigurationBuilder());
```

File: core/src/test/java/org/infinispan/statetransfer/ConcurrentStartTest.java [hunk 2]
```diff
@@ -115,7 +115,7 @@ public void testConcurrentStart() throws Exception {
    private EmbeddedCacheManager createCacheManager(int index) {
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/statetransfer/ConcurrentStartTest.java:115]: nit: `index` isn't used anymore.

File: core/src/test/java/org/infinispan/test/fwk/JGroupsConfigBuilder.java [hunk 3]
```diff
@@ -146,35 +153,40 @@ private static void removeRelay2(JGroupsProtocolCfg jgroupsCfg) {
...
-         portRange = 25;
-         int maxIndex = TCP_PORT_RANGE_PER_THREAD / portRange - 1;
-         if (transportFlags.portRange() > maxIndex) {
-            throw new IllegalStateException("Currently we only support " + (maxIndex + 1) + " ranges/sites!");
-         }
-         startPort += transportFlags.portRange() * portRange;
+      Integer threadIndex = threadTcpIndex.get();
+      int startPort;
+      int portRange;
+      if (siteIndex >= 0) {
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/test/fwk/JGroupsConfigBuilder.java:182]: nit
```suggestion
      if (siteIndex > 0) {
```

File: core/src/test/java/org/infinispan/xsite/statetransfer/ControlledXSiteStateTransferManager.java [hunk 4]
```diff
@@ -12,6 +14,7 @@
  * @since 12.1
  */
 class ControlledXSiteStateTransferManager extends AbstractDelegatingXSiteStateTransferManager {
+   private static final Log log = LogFactory.getLog(SiteUpEvent.class);
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/xsite/statetransfer/ControlledXSiteStateTransferManager.java:17]: why logging under `SiteUpEvent.class`?

File: core/src/test/java/org/infinispan/test/fwk/JGroupsConfigBuilder.java (hunk 3 above)
Comment (danberindei) [reply core/src/test/java/org/infinispan/test/fwk/JGroupsConfigBuilder.java:182]: This is intentional, so we use a smaller port range when siteIndex is set to 0 and the full port range when it's not set.

File: core/src/test/java/org/infinispan/xsite/statetransfer/ControlledXSiteStateTransferManager.java (hunk 4 above)
Comment (danberindei) [reply core/src/test/java/org/infinispan/xsite/statetransfer/ControlledXSiteStateTransferManager.java:17]: Copy paste :)

**Comment by pruivo** [comment]
integrated! thanks @danberindei !

**Pull request merged**